p2p/libp2p/networksharding/errors.go

```diff
@@ -0,0 +1,11 @@
+package networksharding
+
+import (
+	"errors"
+)
+
+var (
+	ErrAlreadySet = errors.New("already set")
```

Comment:
can you keep the same format as in the other errors.go file?

each exported error needs a comment

Reply:
done